README.md

```diff
@@ -38,6 +38,74 @@ Almost all of the implementation was shamelessly adapted from the
 [`stm32f30x-hal`]: https://github.com/japaric/stm32f30x-hal
 [`embedded-hal`]: https://github.com/japaric/embedded-hal
 
+## Selecting the right chip
+
+This crate requires you to specify your target chip as a feature.
+
+*Example: The STM32F3Discovery board has a STM32F303VCT6 chip.
+So you want to expand your call to `cargo` with `--features stm32f303xc`.*
+
+### Possible chips
+
+[comment]: # (Any changes here should be mirrored in src/lib.rs)
```

Comment:
Ha nice trick about the comment. Wouldn't have come to my mind. HTML comments are also valid for markdown

```md
<!-- this is a comment -->
```

Reply:
tjhanks - I made the extra efford to search for it because I wanted pure markdown ;)